template.tpl

```diff
@@ -318,6 +318,44 @@ ___TEMPLATE_PARAMETERS___
         "type": "EQUALS"
       }
     ]
+  },
+  {
+    "type": "GROUP",
+    "name": "additionalInformation",
+    "groupStyle": "ZIPPY_CLOSED",
+    "subParams": [
+      {
+        "type": "SIMPLE_TABLE",
+        "name": "products",
+        "displayName": "Product Information",
+        "simpleTableColumns": [
+          {
+            "defaultValue": "",
+            "displayName": "Product ID",
+            "name": "id",
+            "type": "TEXT",
+            "valueHint": "The item\u0027s ID in the catalog"
+          },
+          {
+            "defaultValue": "",
+            "displayName": "Product Name",
+            "name": "name",
+            "type": "TEXT",
+            "valueHint": ""
+          },
+          {
+            "defaultValue": "",
+            "displayName": "Product Category",
+            "name": "category",
+            "type": "TEXT",
+            "valueHint": "e.g. IAB content taxonomy",
+            "isUnique": false
```

Comment:
Nice!  And another sn00bie question: should we make the product name non-unique as well?  I s'pose I'm thinking that we should have the ID be unique for sure.  But what if one product ID is for a blue tuna can, and another product ID is for a red tuna can, but the product name for both is "tuna can"?

Reply:
Actually, this line isn't supposed to be here at all. I think I accidentally had the advanced settings open when I saved the template. If it's not preset, there's no uniqueness constraint 👍